dist/FeatureCard/FeatureCard.d.ts

```diff
@@ -0,0 +1,51 @@
+/// <reference types="react" />
+export default function FeatureCard(props: {
+    /**
+     * Alternative text to describe the image for screen readers or in situations
+     * where the image cannot be loaded. This prop is required but under certain
+     * circumstances an empty string is preferred.
+     */
+    alt: string;
```

Comment:
I don't think we should support alt text for this component. Since it will almost always be used as a link, a screen reader will read out the alt text and the text content together when the user is focused on the link, which will make it hard to know what the link points to. In `ArticleStrip` we just pass an empty string for the image alt